meetings/2017-01-04.md

```diff
@@ -0,0 +1,331 @@
+copies another property (\_onImmediate) but actually behaves a bit differently
+2-3 months ago. Other similar extraneous properties from similar APIs have been
+removed; this seems to be of the same category.
+
+@jasnell: Any evidence that it’s actually in use?
+
+@trott: Okay with landing this PR, but concerned about hand-wavey nature of
+determining whether something is semver-major.
+
+@jasnell: If it’s in 6 we should take it through a deprecation cycle, if it’s
+only in 7 we can remove immediately.
+
+@mhdawson: If it’s in an LTS it should go through a deprecation cycle.
+
+@jasnell: For clarity to ecosystem it’s better to take these through a
+deprecation cycle.
+
+@Fishrock123: Adding deprecation wrapper may slow down/deopt setImmediate.
+
+@jasnell: If we ensure our code avoids the deprecated property we would avoid
+any deopt or slow down.
+
+@trevnorris: Almost everything in Node is somehow accessible publicly. Does this
+mean that any internal change has to be deprecated first?
+
+@jasnell: We need a better policy to define what must be deprecated.
+
+And perhaps to define what is internal and what is a public API.
+
+@trott: Should we hold on landing this item for the larger discussion?
+
+@jasnell: Because this is in LTS it would be better to go through a deprecation
+cycle.
+
+@trevnorris: Let’s take to a vote in the PR.
+
+**Next steps**:
+
+- Vote in PR.
+- Review James’ issue on deprecation policies
+  [#7964](https://github.com/nodejs/node/pull/7964)
+
+---
+
+### process: Add code to warnings, assign codes to deprecations [#10116](https://github.com/nodejs/node/pull/10116)
+
+Deprecation policy discussion:
+[nodejs/node#7964](https://github.com/nodejs/node/pull/7964)
+
+@jasnell put on agenda to ensure all review and get some LGTM’s.
+
+- Adds a static ID (number) to all deprecations.
+- Adds a runtime flag to redirect deprecation warning output to a file.
+- Also allows us to gather all deprecations in one place in docs.
+
+Goal is to make dealing with deprecations easier.
+
+@Fishrock123: Really old deprecations should have reduced information in docs.
+
+**Next steps**:
+
+- Review this PR.
+
+---
+
+### doc: Deprecate old debug protocol [#10320](https://github.com/nodejs/node/issues/10320)
+
+Doc-only deprecation is also considered semver-major, but we should go ahead and
+land now.
+
+@Fishrock123: Aren’t we violating our own policies in deprecating and removing
+this?
+
+@mhdawson: Because it’s dependent on V8 we’re limited in our options.
+
+@jasnell: Although not ideal, best to include deprecation now in Node 7.
+
+What about runtime deprecation?
+
+@joshgav: A PR is open, needs some updates:
+<https://github.com/nodejs/node/pull/10276>
+
+@trott: Can we include a runtime deprecation before we have `node-inspect`?
+
+@trott: Wait till node-inspect is properly bundled and `node inspect` can
+replace `node debug` till including a runtime deprecation.
```

Comment:
Maybe put a question mark at the end to make it clear that it was more of a question than a statement. Remove line 278 since it repeats this.